themes/classic/_dev/js/product.js

```diff
@@ -104,7 +104,7 @@ $(document).ready(function () {
       max: 1000000
     });
 
-    quantityInput.on('change', function (event) {
+    quantityInput.on('keyup change', function (event) {
```

Comment:
Yes it does only when the input value change :disappointed_relieved: . 
In this case I need them both. Is there an other event that could replace them? (for me I didn't find one)